spec/koans.js

```diff
@@ -163,149 +187,171 @@ describe("destructuring arrays makes shorter code. ", () => {
   it("is simple", () => {
     const obj = { x: 1 };
     /* const {????} = obj */
-    // expect(x).toEqual(1);
+    const { x } = obj;
+    expect(x).toEqual(1);
   });
 
   describe("nested", () => {
     it("multiple objects", () => {
       const magic = { first: 23, second: 42 };
       /* const first, second  = ?????? */
-      // expect(first).toEqual(23);
-      // expect(second).toEqual(42);
+      const { first, second } = magic;
+      expect(first).toEqual(23);
+      expect(second).toEqual(42);
     });
     it("object and array", () => {
       /* const { z: [????] } = { z: [23, 42] } */
-      // expect(x).toEqual(42);
+      const {
+        z: [, x],
+      } = { z: [23, 42] };
+      expect(x).toEqual(42);
     });
     it("array and object", () => {
       /* const [????] = [null, [{ env: "browser", lang: "ES6" }]] */
-      // expect(lang).toEqual('ES6');
+      const [, [{ lang }]] = [null, [{ env: "browser", lang: "ES6" }]];
+      expect(lang).toEqual("ES6");
     });
   });
 
   describe("interesting", () => {
     it("missing refs become undefined", () => {
       /* const {????} = { x: 1, y: 2 } */
-      // expect(z).toBeUndefined();
+      const { z } = { x: 1, y: 2 };
+      expect(z).toBeUndefined();
     });
   });
 });
 
 describe("destructuring can also have default values. ", () => {
   it("for an empty array", () => {
     /* const [????] = [] */
-    // expect(a).toEqual(1);
+    const [a = 1] = [];
+    expect(a).toEqual(1);
   });
 
   it("for a missing value", () => {
     /* const [a, ????, c] = [1, , 3] */
-    // expect(b).toEqual(2);
+    const [a, b = 2, c] = [1, , 3];
+    expect(b).toEqual(2);
   });
 
   it("in an object", () => {
     /* const { a, ???? } = { a: 1 } */
-    // expect(b).toEqual(2);
+    const { a, b = 2 } = { a: 1 };
+    expect(b).toEqual(2);
   });
 
   it("if the value is undefined", () => {
     /* const { a, ???? } = { a: 1, b: undefined } */
-    // expect(b).toEqual(2);
+    const { a, b = 2 } = { a: 1, b: undefined };
+    expect(b).toEqual(2);
   });
 
   it("also a string works with defaults", () => {
     /* const [a, ????] = "1" */
-    // expect(a).toEqual("1");
-    // expect(b).toEqual(2);
+    const [a, b = 2] = "1";
+    expect(a).toEqual("1");
+    expect(b).toEqual(2);
   });
 });
 
 describe("arrow functions. ", () => {
   it("are shorter to write", function () {
     let func = () => {
       /*........*/
+      return "I am func";
     };
-    // expect(func()).toBe("I am func");
+    expect(func()).toBe("I am func");
   });
 
   it("a single expression, without curly braces returns too", function () {
     /*let func = () => .........;*/
-    // expect(func()).toBe('I return too');
+    let func = () => "I return too";
+    expect(func()).toBe("I return too");
   });
 
   it("one parameter can be written without parens", () => {
     /* let func = ........;*/
-    // expect(func(25)).toEqual(24)
+    let func = (number) => number - 1;
```

Comment:
You dont need parenthesis around number:

let func = number => number-1

Reply:
Thx Markus for everything! Yeah I did it without parenthesis but prettier just added them 🤷🏻‍♂️